src/main/scala/uk/gov/nationalarchives/notifications/messages/Messages.scala

```diff
@@ -8,9 +8,15 @@ import io.circe.syntax._
 import sttp.client3.asynchttpclient.cats.AsyncHttpClientCatsBackend
 import sttp.client3.{basicRequest, _}
 import sttp.model.MediaType
-import uk.gov.nationalarchives.aws.utils.Clients.{kms, ses, sns, sqs}
-import uk.gov.nationalarchives.aws.utils.SESUtils.Email
-import uk.gov.nationalarchives.aws.utils.{KMSUtils, SESUtils, SNSUtils, SQSUtils}
+import uk.gov.nationalarchives.aws.utils.kms.KMSClients.kms
```

Comment:
Maybe use the `.{ ... }`and `._` import syntax to reduce the number of individual import statements?

Reply:
It's a bit tricky because they've all got different package names but I've changed it to import everything like `import uk.gov.nationalarchives.aws.utils.ses._` and then prefixed the client like `SESClient.ses` which I think makes things clearer as it's obvious where the client is coming from.